CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [0.12.3+1]
```

Comment:
Please update the version in the `pubspec.yaml` to match this.

Reply:
Done